### ISSUE TITLE
Debian-based Postgres

### ISSUE THREAD

**Issue by GeoTob** [kubedb/project#487]
I've been playing around with creating a Debian-based Postgres image. This is mainly to have access to `glibc` for various extensions that I couldn't get to work nicely on Alpine.

Got it to a stage where I can run a cluster and verify that replication works as expected, and hoping someone here can help me check my approach. 

Dockerfile: https://gist.github.com/GeoTob/8a9a2ee262e9fbf649e92c7f35f07593

I'm a bit worried that I've missed something critical, since kubedb dependencies may have been compiled against / tuned for Alpine. I'd like to be able to keep this up-to-date with future KubeDB versions and patches -- also happy to contribute my Dockerfile to the project if you're interested in rolling it into KubeDB/postgres

**Comment by david-ecc** [comment]
See: https://hub.docker.com/_/postgres
There are Debian and Alpine-based images

**Comment by GeoTob** [comment]
True, could've just grabbed the official Postgres-debian - thanks.

My question is more around installing the KubeDB scripts & binaries. Is the approach used in the Dockerfile OK ?

**Comment by david-ecc** [comment]
This Dockerfile works for me:
```
FROM postgres:11.2

RUN apt-get update && apt-get -y install gosu && rm -rf /var/lib/apt/lists/* && ln -s gosu /usr/local/bin/su-exec

ENV PV /var/pv
ENV PGDATA $PV/data
ENV PGWAL $PGDATA/pg_wal
ENV INITDB /var/initdb
ENV WALG_D /etc/wal-g.d/env

COPY --from=kubedb/postgres:11.1-v1 /usr/bin/pg-operator /usr/bin/
COPY --from=kubedb/postgres:11.1-v1 /usr/bin/wal-g /usr/bin/
COPY --from=kubedb/postgres:11.1-v1 /scripts /scripts

VOLUME ["$PV"]

ENV STANDBY warm
ENV RESTORE false
ENV BACKUP_NAME LATEST
ENV PITR false
ENV ARCHIVE_S3_PREFIX ""
ENV RESTORE_S3_PREFIX ""

ENV ARCHIVE_GS_PREFIX ""
ENV RESTORE_GS_PREFIX ""

ENTRYPOINT ["pg-operator"]
CMD ["leader_election"]

EXPOSE 5432
```
